### PR TITLE
let me turn on debugging

### PR DESCRIPTION
**Pull request by saolsen**
because it's broke

### CODE REVIEW

File: config/initializers/datadog.rb [hunk 1]
```diff
@@ -3,9 +3,10 @@
 Datadog.configure do |c|
   enabled = ENV.fetch('DATADOG_ENABLED', false)
   hostname = ENV.fetch('TRACE_AGENT_HOSTNAME', 'localhost')
+  debug = ENV.fetch('DATADOG_DEBUG', false)
```
Comment (ashkan18) [inline config/initializers/datadog.rb]: one thing though! does `debug` flag down in tracer accept string? basically these `ENV.fetch` will return String and not boolean.

**Comment by orta** [comment]
You merge on green'd in a github review (not a comment) you should add support - https://github.com/artsy/peril-settings/issues/81

File: config/initializers/datadog.rb (hunk 1 above)
Comment (saolsen) [reply config/initializers/datadog.rb]: oh maybe that's the whole problem, what's the move to parse as a bool?

Comment (ashkan18) [reply config/initializers/datadog.rb]: yes your `== 'true'` should work

**Pull request merged**